config/routes.rb

```diff
@@ -13,6 +15,9 @@
   # end
   resources :placements, :comments, :licensees
   resources :placements_imports, only: [:new, :create]
+  resources :licensees_imports, only: [:new, :create]
```

Comment:
This seems redundant with lines 2 and 3 above. 

Please delete lines 2-5 and keep the resource versions here.